### ISSUE TITLE
Failed to get IP for distro

### ISSUE THREAD

**Issue by robertgeb** [shayne/go-wsl2-host#15]
Hi,
Thanks for sharing this tool.
I try to run using Arch and get this error runing 'debug'
"failed to get IP for distro "Arch": exit status 64"
As I didn't find anyone with the same problem and I don't have the slightest idea of ​​what could be, I decided to open this issue.

**Comment by GoodbyeNJN** [comment]
Same problem for me at ArchWsl.
I read the code and found the method of getting wsl ip at:
https://github.com/shayne/go-wsl2-host/blob/f76c851a2464aba0d29b48117d21b8fe05f3a5f8/pkg/wslcli/wslcli.go#L48
So I tried to install `inetutils` package which provides `hostname` command, but this `hostname` command does not support argument `-I`.
Finally I found a workaround, I downloaded debian package `hostname` from [https://packages.debian.org/buster/hostname](https://packages.debian.org/buster/hostname), extract it and copy `hostname` file to `/usr/bin/hostname`.
Things seem all right and the method of getting wsl ip works fine, but this still can course some unknown problem and the best way is to modify the method of getting wsl ip.
